src/GitTfs/Commands/RemoteOptions.cs

```diff
@@ -24,6 +24,8 @@ public OptionSet OptionSet
                         v => Username = v },
                     { "p|password=", "TFS password",
                         v => Password = v },
+                    { "pat=", "TFS Personal Access Token",
```

Comment:
This would require the corresponding documentation changes in e.g.
https://github.com/git-tfs/git-tfs/blob/master/doc/commands/clone.md#authentication

and also updates in the synopsis of the following commans. 
https://github.com/git-tfs/git-tfs/blob/master/doc/commands/clone.md
https://github.com/git-tfs/git-tfs/blob/master/doc/commands/init.md
https://github.com/git-tfs/git-tfs/blob/master/doc/commands/fetch.md
https://github.com/git-tfs/git-tfs/blob/master/doc/commands/pull.md
https://github.com/git-tfs/git-tfs/blob/master/doc/commands/list-remote-branches.md

Reply:
I can make those changes if you want, and submit them in another PR, if you approve this one. Or I can make edits to this PR. Not sure which way you prefer to go.

Reply:
Please make them to this PR